drivers/storage/ide/atapi/include/debug.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * PROJECT:     ReactOS ATA Port Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Debug support header file
+ * COPYRIGHT:   Copyright 2024-2025 Dmitry Borisov <di.sean@protonmail.com>
+ */
+
+#pragma once
+
+#ifndef __RELFILE__
+#define __RELFILE__ __FILE__
+#endif
+
+#include <string.h>
+#define __FILENAME__ (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)
+
+#define My_DbgPrint DbgPrint
+
+#if DBG
+
+// #define DEBUG_TRACE
+#define DEBUG_INFO
+#define DEBUG_INFO_VERB
+#define DEBUG_WARN
+#define DEBUG_ERR
+// #define DEBUG_TRACE_IO
+
+#ifdef DEBUG_TRACE
+#define TRACE(fmt, ...) \
+    do { \
+      if (My_DbgPrint("ATAPI: (%s:%d) %s " fmt, __FILENAME__, __LINE__, __FUNCTION__, ##__VA_ARGS__)) \
+          My_DbgPrint("(%s:%d) DbgPrint() failed!\n", __RELFILE__, __LINE__); \
+    } while (0)
+
+#else
+#define TRACE
```

Comment:
Shouldn't this rather be
```suggestion
#define TRACE(fmt, ...)
```

Reply:
Fixed.